ui/00_message/common/item_info.xml

```diff
@@ -70,18 +71,30 @@ Elven characters are inscribed.</translation>
   <item id="48">
     <original>ガルドノック砦のどこかの鍵
 金属部には血色の錆びが浮いている</original>
+    <translation>A key used somewhere in Gardonox Fortress.
+      Blood is rusting the metal.</translation>
   </item>
   <item id="49">
     <original>ドリード《城主の間》の扉を開く鍵
 白竜と盾を模した文様が刻まれている</original>
+    <translation>A key to open the Master Chamber in Dreed Castle.
+      A shield and a white dragon are engraved.</translation>
+      <!-- Needs Review -->
+      <!--This may be wrong, as far as I can tell this is the most likely translation for
+      ドリード《城主の間》の扉を開く鍵 but Im not sure. If someone has played the quest they can verify this
+      -->
   </item>
   <item id="50">
     <original>ドリード城地下、最深部《監獄牢》の鍵
 収監された者がどうなったかは不明</original>
+    <translation>A key to the prison underneath Dreed Castle.
+      You don't know what happened to the prisioners</translation>
   </item>
   <item id="52">
     <original>投げつけた対象にダメージと
 状態異常【延焼】を与える</original>
+    <translation>Throw this to cause damage on impact.
+      Causes the [Burning] debilitation.</translation>
```

Comment:
Better to use quotes instead of brackets when reformatting "【】" in this situation.